tests/test_dependencies.py

```diff
@@ -35,14 +41,18 @@ def test_bioconda() -> None:
     assert error_code == 0, stderr
 
 
+@pytest.mark.skipif(not deps, reason="galaxy-lib is not installed")
 @pytest.mark.skipif(
     not spawn.find_executable("modulecmd"), reason="modulecmd not installed"
 )
-def test_modules() -> None:
+def test_modules(monkeypatch: Any) -> None:
+    """Here we do a basic smoke test using environment modules to
+    satisfy a SoftwareRequirement.
+    """
```

Comment:
```suggestion
    """Do a basic smoke test using environment modules to satisfy a SoftwareRequirement."""
```

To fix https://travis-ci.com/github/common-workflow-language/cwltool/jobs/454295930#L4188